client/src/App.scss

```diff
@@ -0,0 +1,29 @@
+@import "./Utilities/SCSS-Variables/variables.scss";
+
+* {
+  font-family: "Dosis";
+}
```

Comment:
create a reset.scss file and add reset code like margin 0, padding 0 to it. Add font family to the global body rule, not here